src/main/java/lc/MainCaptcha.java

```diff
@@ -0,0 +1,23 @@
+package lc;
+
+public class MainCaptcha implements ChallengeProvider{
+
+    public String getId(){
+        return "SomeText";
+    }
+
+    public Challenge returnChallenge(){
+        CaptchaTypes captchaTypes = new CaptchaTypes();
+        return new Challenge(captchaTypes.type_1("Hello"),"png","qwert");
```

Comment:
You need to create a random string and use that for creating the captcha image, and also return the same string as the `secret`.